coral/src/app/features/topics/list/TopicCard.module.css

```diff
@@ -0,0 +1,6 @@
+/*@TODO use aiven-ds based values*/
+.topic-card-title-desc {
+    border-bottom: 3px solid orange;
+    padding: 10px 0;
+    margin-bottom: 10px;
+}
```

Comment:
I think the comment refers to an older commit, because my state has another class (and a newline at the end)? Will check after committing the changes!

Reply:
Most likely. I commented this while reading the changed one commit at a time.